src/engine/sparqlExpressions/SparqlExpressionValueGetters.h

```diff
@@ -168,6 +170,18 @@ struct LiteralFromIdGetter {
   }
 };
 
+struct RegexValueGetter {
+  std::unique_ptr<re2::RE2> operator()(SingleExpressionResult auto&& input,
+                                       const EvaluationContext* context) const
+      requires requires { StringValueGetter{}(AD_FWD(input), context); } {
```

Comment:
TODO<joka921> Comment the regexValueGetter and maybe move it somewhere.
also TODO<joka921> Can we implement the `Ordinary` regex expression as an `NARY-Operation` with specialized functions?